pytorch_lightning/plugins/training_type/horovod.py

```diff
@@ -96,22 +96,22 @@ def start_training(self, trainer):
                 stack.enter_context(optimizer.skip_synchronize())
 
             # set up training routine
-            self._results = trainer.run_train()
+            self._results = trainer.run_stage()
```

Comment:
Shall this stage be also an argument? so it is cleaner?

Reply:
what do you mean ? In my perspective, we should always call `run_stage`, so we are sure distributed hook can be called properly.

Reply:
We generally don't pass these arguments and instead rely on the trainer attributes. So i'd say no